common/lib/xmodule/xmodule/x_module.py

```diff
@@ -35,6 +35,7 @@
 from xmodule.exceptions import UndefinedContext
 import dogstats_wrapper as dog_stats_api
 
+from xmodule import course_metadata_utils
```

Comment:
I'll move the import up.

But the reason I like importing the entire module is that it makes all the functions calls qualified (i.e. `course_metadata_utils.do_something()` vs just `do_something()`) which I think is very helpful in providing context, especially in this type of scenario where we're introducing a bunch of indirection with these utility functions.